Add new test and fix for 3309

willwendorfvm opened the pull request:
Test that conf_remap can override float settings (e.g. proxy.config.http.background_fill_completed_threshold) successfully

SolidWallOfCode commented on tests/gold_tests/autest-site/trafficserver.test.ext:
```diff
@@ -221,6 +221,11 @@ def MakeATSProcess(obj, name, command='traffic_server', select_ports=True, enabl
     p.Disk.File(tmpname, id=make_id(fname), typename="ats:config")
 
+    # for describe-float, used by conf_remap plugin
+    fname = "mysterio.config"
```
Can't use that name, it's internal. Use something like "delain.config". "Delain" is *always* a good choice for names in examples.

willwendorfvm replied:
Changed to delain.

SolidWallOfCode commented on tests/gold_tests/remap/conf_remap_float.py:
```diff
@@ -0,0 +1,52 @@
+import os
+
+Test.Summary = '''
+Test command: sudo traffic_ctl config describe proxy.config.http.background_fill_completed_threshold (YTSATS-3309)
```
It should not be necessary to `sudo` if TS is started up as a normal user.  Susan also says that `config describe` shouldn't need `sudo` in any case.

willwendorfvm replied:
Removed sudo. I think that was a typo.

SolidWallOfCode reviewed (commented):
Looks good with a few changes.

willwendorfvm committed:
Add new test and fix for 3309

Test that conf_remap can override float settings (e.g. proxy.config.http.background_fill_completed_threshold) successfully.

SolidWallOfCode commented:
Why did you merge this in to your master?

willwendorfvm commented:
Thought it was normal when closing. The PR back to the open source repo is still off the branch, though.
